qemu/tests/cfg/rng_stress.cfg

```diff
@@ -15,20 +15,21 @@
         driver_id_pattern = "(.*?):.*?VirtIO RNG Device"
     Linux:
         session_cmd_timeout = 360
-        read_rng_cmd  = "dd if=/dev/hwrng  bs=1 count=10 2>/dev/null|hexdump"
+        read_rng_cmd  = "dd if=/dev/urandom bs=1 count=10 2>/dev/null|hexdump"
```

Comment:
shuang, it should read the /dev/random device instead of /dev/urandom; for /dev/urandom device, it's used as virtio rng backend in host.